ParcelDistributionCenter/ParcelDistributionCenter.Model/Context/Seed.cs

```diff
@@ -29,7 +29,7 @@ public void Initialize(ParcelDistributionCenterContext context)
             List<Courier> couriers = _memoryRepository.CouriersList;
             foreach (Courier courier in couriers)
             {
-                IEnumerable<Package> courierPackages = packages.Where(p => p.CourierJsonId == courier.CourierJsonId);
+                IEnumerable<Package> courierPackages = packages.Where(p => p.CourierId == courier.CourierJsonId);
```

Comment:
Czy tutaj na pewno tak będzie? CourierJsonId nadałem, żeby zmapowało te jsonowe idiki, czyli one się przydadzą tylko do Seeda. Tak samo wyżej. Aż tyle czasu teraz nie mam, ale trzeba się zastanowić, kiedy powinno być które Id :D

Reply:
Moim zdaniem powinno tak zostać. 
Jak zostawimy to "CourierJsonId" w modelu "Package", to potem dalej w metodach "CourierConttroller" nam się w wielu miejscach będzie pojawiało to "CourierJsonId" a tak być nie powinno.
Zmieniłem tą nazwę zarówno w modelu "Package" jak i w "Seed", więc wszystko powinno zadziałać.